src/app/authentication/models/signUpModel.go

```diff
@@ -1 +1,12 @@
 package models
+
+type Customer struct {
+	ID            int    `json:"id"`
+	Name          string `json:"name" validate:"required"`
+	Email         string `json:"email" validate:"required"`
+	PhoneNumber   string `json:"phoneNumber" validate:"required,len=10"`
+	PanCardNumber string `json:"panCardNumber" validate:"required"`
+	Password      string `json:"password" validate:"required"`
+}
```

Comment:
Use inbuilt validator refer https://pkg.go.dev/github.com/go-playground/validator/v10#section-readme
Also register custom validator and use here.